pkgs/development/python-modules/vqgan-jax/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flax
+, jaxlib
+, pytorch
+, transformers
+}:
+
+buildPythonPackage rec {
+  pname = "vqgan-jax";
+  version = "unstable-2022-04-20";
+
+  src = fetchFromGitHub {
+    owner = "patil-suraj";
+    repo = "vqgan-jax";
+    rev = "1be20eee476e5d35c30e4ec3ed12222018af8ce4";
+    sha256 = "sha256-OZihAXpE0UsgauQ38XDmAF+lrIgz05uK0ro8SCdVsPc=";
+  };
+
+  propagatedBuildInputs = [
+    flax
+    jaxlib
+    transformers
+  ];
+
+  checkInputs = [
+    pytorch
+  ];
+
+  meta = with lib; {
+    description = "JAX implementation of VQGAN";
+    # license unknown: https://github.com/patil-suraj/vqgan-jax/issues/9
```

Comment:
```suggestion
    # license unknown: https://github.com/patil-suraj/vqgan-jax/issues/9
    license = lib.licenses.unfree;
```

Reply:
Is this actually correct? The package is not necessarily unfree. It just doesn't have a license. I think leaving out the license is more accurate to reflect that.

Reply:
In most jurisdictions the lack of a license means that the auther has all rights, so it's unfree.